### ISSUE TITLE
Delete Unpublished Request

### ISSUE THREAD

**Issue by Jshcwn** [Attention-Offer-Networks/AON#3]
As an End User I would like to delete an unpublished request so I can choose not to publish a request I had already started

**Comment by edoubt** [comment]
Is this necessary?  I don't see any other steps that specifically require "publishing" a request (vs. simply completing it).  Alternatively maybe this should just be "Delete a published request"

**Comment by Jshcwn** [comment]
My thinking was that the end user might start a request but, prior to finishing it, want to delete the request. This is part of my assumption that they'd want to have the ability to work on a request but not finish it in one session and/or then choose whether to show this request to the public.

**Comment by edoubt** [comment]
Makes sense to me, but I think then this workflow necessitates at least one (if not several) more step(s) being iterated as issues: 
1.  Publish request
2.  Save draft of unpublished request
3.  ?

**Comment by Jshcwn** [comment]
Yup, makes sense. I suspect this will be clearer when we know how extensive a request can be in the prototype. If it is an in depth document they will need "saving" and "publishing" capabilities. If it is a small enough, then my concerns may be unwarranted. I would leave this up in the air for the moment.
